tests/basic/storage_pool_test.go

```diff
@@ -12080,3 +12081,234 @@ var _ = Describe("{PoolDeleteMultiplePools}", func() {
+				for _, eachVol := range vols {
+					volDetails := VolumeDetails{}
+
+					// Get details on the Volume
+					volDetails.vol = eachVol
+
+					// Get PVC object from the Volume
+					pvc, err := GetPVCObjFromVol(eachVol)
+					log.FailOnError(err, "Failed to get PVC Details from Volume [%v]", eachVol.Name)
+
+					// Update volSizeMap to perform Parallel Resize of all the Volumes present in the cluster
+					volDetails.pvc = pvc
+					volSizeMap = append(volSizeMap, &volDetails)
+				}
+			}
+		})
+
+		for _, eachVol := range volSizeMap {
+			// Pod details after blocking IP
+			podsAfterblk, err := k8sCore.GetPodsUsingPVC(eachVol.vol.Name, eachVol.vol.Namespace)
+			log.FailOnError(err, "unable to find the node from the pod")
+			for _, eachPodAfter := range podsAfterblk {
+				log.Infof(fmt.Sprintf("Current state of the Pod [%v] is [%v]", eachPodAfter.Name, eachPodAfter.Status.Phase))
+				if eachPodAfter.Status.Phase != "Running" {
+					log.Infof(fmt.Sprintf("State of the Pod before Resize volume [%v] is [%v]", eachPodAfter.Name, eachPodAfter.Status.Phase))
+					log.FailOnError(fmt.Errorf("Pod [%v] Consuming Volume [%v] is not in Running State ",
+						eachPodAfter.Name, eachVol.vol.Name), "Pod not in Running State")
+				}
+			}
+		}
+
+		// Volume resize routine resizes specific Volume
+		log.InfoD("start Volume resize on each Volume present in the cluster")
+		ResizePvcInParallel := func(newVolumeIDs *VolumeDetails) {
+			defer GinkgoRecover()
+			volCurSize := newVolumeIDs.vol.Size / units.GiB
+			volNewSize := volCurSize + uint64(10)
+			log.Infof("Resizing Volume [%v] from size [%v] to [%v]", newVolumeIDs.vol.Name, newVolumeIDs.vol.Size, volNewSize*units.GiB)
+			vol, err := Inst().S.ResizePVC(newVolumeIDs.ctx, newVolumeIDs.pvc, volNewSize)
+			log.FailOnError(err, "Failed to resize PVC [%v]", vol.Name)
+
+			time.Sleep(60 * time.Second)
+			volReSize, err := Inst().V.InspectVolume(vol.ID)
+			log.FailOnError(err, "inspect returned error ?")
+
+			log.Infof("Verify volume size after resizing the volume [%v]", vol.Name)
+			dash.VerifyFatal(volReSize.Spec.Size > newVolumeIDs.vol.Size, true, "Resize of Volume didnot happen!")
+			log.Infof(fmt.Sprintf("Volume [%v] resized from [%v] to [%v]", newVolumeIDs.vol.Name, newVolumeIDs.vol.Size, volReSize.Spec.Size))
+		}
+
+		// Resize Volumes in parallel
+		ResizeVolumes := func(volId string) {
+			defer GinkgoRecover()
+			sizeBeforeResize, err := Inst().V.InspectVolume(volId)
+			log.FailOnError(err, "inspect returned error ?")
+			toResize := sizeBeforeResize.Spec.Size + (10 * units.GiB)
+
+			// Resize Volume and verify if volume resized
+			log.FailOnError(Inst().V.ResizeVolume(volId, toResize), "Failed to resize volume")
+
+			//Size after resize
+			sizeAfterResize, err := Inst().V.InspectVolume(volId)
+			log.FailOnError(err, "inspect returned error ?")
+
+			dash.VerifyFatal(sizeBeforeResize.Spec.Size < sizeAfterResize.Spec.Size, true, "Volume resize did not happen")
+
```

Comment:
Same comment as above - let's also add check for the newly resized size apart from less than or greater than previous size

Reply:
done